docs/How To/Change filter based on time of day.md

```diff
@@ -0,0 +1,25 @@
+---
+publish: true
+---
+
+# Change filter based on time of day
+
+<span class="related-pages">#css</span>
+
+This example provides a way to change the filter being run based on the time of day.
+
+    ```task-sql
+    WHERE status->indicator = ' ' AND (
+      moment()->isBetween(moment('08:00:00','hh:mm:ss'), moment('17:00:00','hh:mm:ss')) AND tags->includes('#Work')
+    OR
+      NOT moment()->isBetween(moment('08:00:00','hh:mm:ss'), moment('17:00:00','hh:mm:ss')) AND tags->includes('#Home')
+    )
+    ```
+
+```task-sql
+WHERE status->indicator = ' ' AND (
```

Comment:
Nice. This is another one that needs its tag updated and de-duplicating the example.